src/audio/eq_fir/eq_fir.c

```diff
@@ -151,24 +142,24 @@ static inline int set_fir_func(struct comp_dev *dev)
 	switch (sourceb->stream.frame_fmt) {
 #if CONFIG_FORMAT_S16LE
 	case SOF_IPC_FRAME_S16_LE:
-		trace_eq_with_ids(dev, "set_fir_func(), SOF_IPC_FRAME_S16_LE");
+		comp_info(dev, "set_fir_func(), SOF_IPC_FRAME_S16_LE");
 		set_s16_fir(cd);
```

Comment:
@mmaka1 Would be nice if we could retrieve component class from dev. Thus, this call would just transform to:
```
dev_info(dev, "set_fir_func()...");

Reply:
Good catch, I think adding a new field with the component class can be done in here, but my thoughts are about the componentless traces (comp_cl_info etc)

Reply:
So as far as I understand the component from SOF is the equivalent of dev from Linux kernel. We can have two sets of tracing macros:

* for components

```
#define dev_emerg(dev, fmt, ...)
#define dev_crit(dev, fmt, ...)
#define dev_alert(dev, fmt, ...)
#define dev_err(dev, fmt, ...)
#define dev_warn(dev, fmt, ...)
#define dev_notice(dev, fmt, ...)
#define dev_info(dev, fmt, ...)
```
* for componentless trace we could use an equivalent of pr_* from the kernel

```
#define pr_emerg(dev, fmt, ...)
#define pr_crit(dev, fmt, ...)
#define pr_alert(dev, fmt, ...)
#define pr_err(dev, fmt, ...)
#define pr_warn(dev, fmt, ...)
#define pr_notice(dev, fmt, ...)
#define pr_info(dev, fmt, ...)
#define pr_debug(dev, fmt...)
```

We could use our own convention: e.g comp_info instead of dev_info and trace_info instead of pr_info.

Reply:
@dbaluta Yeah but still the equivalent of the pr_* traces still need to contain a trace class (there isn't a TRACE_CLASS_NULL as far as I know; just move them under the generic component trace class?). That would still be a behavior change and might even affect which traces are enabled vs which are disabled.